### PR TITLE
Landing Page and README edits

### PR DESCRIPTION
**Pull request by shiffman**
I've made some language changes to the landing page and README. Although I think maybe the stuff I added to the README should go in CONTRIBUTING?

### CODE REVIEW

**Comment by shiffman** [comment]
Doh I should really learn how to use git.

**Comment by shiffman** [comment]
Ok, I think this is set now. Will definitely need a squash!

**Comment by shiffman** [comment]
Heh, I really messed something up here.

**Comment by nilsweb** [comment]
Nothing that a good old "Squash and Merge" couldn't fix! 😜

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,3 +22,23 @@ If you are unsure about git and GitHub, and how to make your contribution,
 these [Git and GitHub for Poets tutorials](https://www.youtube.com/playlist?list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV) might help.
 
 Happy Coding!
+
+## Website Development
```
Comment (nilsweb) [inline CONTRIBUTING.md]: I've written a quick [guide](https://github.com/CodingTrain/website/wiki/Core-Contribution-Guide) on how to set up a local dev environment. Maybe we should add a link to it here?

File: landing-page.html [hunk 2]
```diff
@@ -54,12 +54,21 @@ <h2>Latest Videos</h2>
   <h2>Support The Coding Train</h2>
   <div class="support">
     {% capture patreonLogo %}{% include 4-vector-graphics/patreon.svg %}{% endcapture %}
-    {% include 3-modules/hero-item.html image=patreonLogo title='Consider supporting?' content='Support The Coding Train on Patreon and get some awesome rewards!' url=site.links.patreon cta='Become a Patron' class='patreon' %}
+    {% include 3-modules/hero-item.html image=patreonLogo title='Consider supporting?' content='Join the Coding Train
+    community with a YouTube membership or Patreon subscription!'
```
Comment (nilsweb) [inline landing-page.html]: I'll test if you can have a link here but it should work by just including HTML... 🤔

File: landing-page.html [hunk 3]
```diff
@@ -12,17 +12,17 @@
 
 {% comment %} Use the latest coding challenge if a stream hasn't been scheduled yet. {% endcomment %}
 {% if nextStream == null %}
-  {% assign offset = 1 %}
```
Comment (nilsweb) [inline landing-page.html]: I think your prettifier wasn't happy with my formatting... 😭 We should keep it indented for better readability!

File: CONTRIBUTING.md (hunk 1 above)
Comment (shiffman) [reply CONTRIBUTING.md]: Yes, of course, sorry didn't realize that the jekyll instructions were there also, this is redundant. I'll adapt!

File: landing-page.html (hunk 2 above)
Comment (shiffman) [reply landing-page.html]: Hah, so when I added the HTML links this is what happened. I guess we need a different style/class for inline?

screen shot 2018-10-21 at 3 04 14 pm

Comment (shiffman) [reply landing-page.html]: Going to take the links out for now so this can be merged and we can sort out as a separate issue?

Comment (nilsweb) [reply landing-page.html]: Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Comment (nilsweb) [reply landing-page.html]: Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Comment (nilsweb) [reply landing-page.html]: Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Comment (nilsweb) [reply landing-page.html]: Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Comment (nilsweb) [reply landing-page.html]: Oh... That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Comment (nilsweb) [reply landing-page.html]: Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Comment (nilsweb) [reply landing-page.html]: Oh... 🤔 That's definitely not right. I'll fix it in a separate PR so we can merge this now!

Comment (nilsweb) [reply landing-page.html]: Oh... 🤔 I'll sort this out in a separate PR so we can merge this one.

Comment (nilsweb) [reply landing-page.html]: Is this visible?

**Comment by nilsweb** [comment]
Since GitHub seems to have some issues right now, I'll write it here:
The links shouldn't look like this... 😂 I'll sort it out in a separate PR so we can merge this one! 👍🏻

File: landing-page.html (hunk 2 above)
Comment (shiffman) [reply landing-page.html]: weird, your comment came in 6 times!

**Comment by shiffman** [comment]
Merging!

**Pull request merged**